iphone/TitaniumKit/TitaniumKit/Sources/API/ObjcProxy.m

```diff
@@ -279,6 +280,12 @@ - (void)_fireEventToListener:(NSString *)type withObject:(id)obj listener:(JSVal
 
   if (listener != nil) {
     [listener callWithArguments:@[ eventObject ]];
+    // handle an uncaught exception
+    JSValue *exception = [[listener context] exception];
```

Comment:
I prefer to use property accessors when possible to avoid all those square brackets.

```suggestion
    JSValue *exception = listener.context.exception;
```